### DISCUSSION TITLE
export MIME types are all of type 'text/html'

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#15778] [Help]
# Bug report

## Describe the bug

Exporting static builds ends up with setting 'text/html' on css files, which in turn circumvents the rendering of these files.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Create a new Next app
2. Run the build script
3. Run the export script
4. Use something like `npx serve ./out` to serve the contents of the /out folder

You'll notice in the console a bunch of MIME type errors when relating to css. 

Issue doesn't seem to happen when using a node-sass build.

## Expected behavior

The export script should correctly generate MIME types for each file type

## System information

- Version of Next.js: 9.5.1

**Comment by timneutkens** [comment]
If you're going to run `npx serve` it's better to just use `next start` instead.

> The export script should correctly generate MIME types for each file type

Static files do not have mime types and they can't be set. They have the correct extension though so I highly doubt this is a Next.js issue, will convert this to a help discussion
